src/utils/facilities/formatFacilityToServer.spec.js

```diff
@@ -0,0 +1,37 @@
+import omit from 'lodash.omit';
+import formatFacilityToServer from './formatFacilityToServer';
+
+describe('utils/facilities/formatFacilityToServer', function() {
+  let expectedFacility;
+  let facility;
+  let formattedFacility;
+
+  beforeEach(function() {
+    facility = fixture.build('facility');
+    expectedFacility = omit(
```

Comment:
It feels like this approach is frail and can be broken fairly easily by unrelated changes, but I have no suggestions on how to improve it.

Reply:
I don't really like it much either but haven't come up with a great way to handle it. Tbh, even coming up with doing it this way was kind of a challenge. If you think of a good idea, lmk